style.css

```diff
@@ -0,0 +1,47 @@
+h1 {
+  font-size: 200px;
+  color: white;
+ 
+  
+}
+
+h2 {
+  color: white;
+  text-align: left;
+ 
+   
+
+}
+body {
+  
+  font-family: "Helvetica Neue", sans-serif;
+  font-size: 24px;    
+  
+  
+}
```

Comment:
Your `body` selector and ruleset should start the stylesheet; this sets the defaults for all other elements.